website/scriptFactory.py

```diff
@@ -1,57 +1,71 @@
-#Generic for script.cel file
-script = [
-'{\n',
-	'time{utc "2015-01-01T00:01:24.0000"}\n',
-	'select{object "Orbit-test-spacecraft"}\n',
-	'center {time 5.0}\n',
-	'goto { time 5.0 }\n',
-	'follow{}\n',
-	'lock{}\n',
-'}'
-]
-
-#Generic for orbit.ssc file
-scriptSSC = [
-'"Orbit-test-spacecraft" "Sol"\n',
-'{\n',
-'\tClass "spacecraft"\n',
-'\tMesh "orbit.3ds"\n',
-'\tRadius 0.011\n',
-'\tOrientation [ 180 1 0 0 ]\n',
-'\tTimeline [\n',
-'\t# Phase 3: Solstice mission\n',
-'\t{\n',
-'\tBeginning "2015 01 01 00:00:00"\n',       
-'\tEnding    "2020  9 15 17:02:00"\n',
-'\tOrbitFrame { EclipticJ2000 { Center "Sol/Earth" } }\n',
-'\tSampledTrajectory { Source "orbit.xyzv" }\n',
-'\t}\n',
-'\t]\n',
-'}\n',
-]
-
-
-def makeScript(beginDate,shipName):
+from datetime import date, datetime
+
+def makeScript(shipName="ORBIT", beginDate=date.today().year):
+	script = [
+	'{\n',
+		'time{utc "2015-01-01T00:01:24.0000"}\n',
+		'select{object "Orbit-test-spacecraft"}\n',
+		'center {time 5.0}\n',
+		'goto { time 5.0 }\n',
+		'follow{}\n',
+		'lock{}\n',
+	'}'
+	]
```

Comment:
if(beginDate==null)
{
beginDate=DateTime.Now.Year
}
	script = [
 -	'select{object "Orbit-test-spacecraft"}\n',		 +	'{\n',
 -	'center {time 5.0}\n',		 +		'time{'+beginDate+'}\n',
 -	'goto { time 5.0 }\n',		 +		'select{object "Orbit-test-spacecraft"}\n',
 -	'follow{}\n',		 +		'center {time 5.0}\n',
 -	'lock{}\n',		 +		'goto { time 5.0 }\n',
 -'}'		 +		'follow{}\n',
 -]		 +		'lock{}\n',
 -		 +	'}'
 -#Generic for orbit.ssc file		 +	]